geode-core/src/main/java/org/apache/geode/distributed/internal/locks/DLockService.java

```diff
@@ -787,39 +783,40 @@ private void notLockGrantorId(LockGrantorId notLockGrantorId, long timeToWait,
       if (logger.isTraceEnabled(LogMarker.DLS_VERBOSE)) {
```

Comment:
It is intentional to avoid that trap. In the past we have been bad about changing units of time and not correcting the math at all the call points. While all the current usages always use milliseconds that may not be the case in the future. Time is hard and has bit us in the rear a few too many times.